gulp-tasks/check/links.js

```diff
@@ -0,0 +1,14 @@
+const {UrlCrawl} = require('webdev-infra/utils/url-crawl');
+const handlers = require('../../server/handlers');
+
+new UrlCrawl({
```

Comment:
I think for this to be a proper gulp task you would need to wrap this in an (async) function and then export it from here. Also move to just gulp-tasks/checkLinks.js, since we only have one check task so far.